client/express-checkout/_spinner.scss

```
@@ -0,0 +1,26 @@
+	content: '';
+	display: block;
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	width: 1em;
+	height: 1em;
+	margin-top: -0.5em;
+	margin-left: -0.5em;
+	font-size: 2em;
+	box-sizing: border-box;
+	border: 0.2em solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)